src/schema/v2/conversation/collectorResume.ts

```diff
@@ -25,5 +26,26 @@ export const CollectorResume = new GraphQLObjectType<any, ResolverContext>({
         "Collector's ID used to stitch buyerActivity with the Exchange schema",
       resolve: ({ userId }) => userId,
     },
+    purchases: {
+      type: CollectorPurchasesType,
+      description: "non-bnmo Collector's purchase history",
+      resolve: ({ purchases }) => purchases,
+    },
   }),
 })
+
+const CollectorPurchasesType = new GraphQLObjectType<any, ResolverContext>({
+  name: "purchases",
+  fields: {
+    totalAuctionCount: {
+      type: GraphQLNonNull(GraphQLInt),
+      description: "Total number of auction winning bids",
+      resolve: ({ auction }) => auction || 0,
+    },
+    totalPrivateSaleCount: {
+      type: GraphQLNonNull(GraphQLInt),
+      description: "Total number of private sales",
+      resolve: (data) => data["private sale"] || 0,
```

Comment:
**nitpick:** Can you follow the pattern set above and do `resolve({ private_sale }) => private_sale || 0`?

Reply:
Unfortunately not - the field is coming from the [Purchase type "private sale"](https://github.com/artsy/gravity/blob/main/app/models/domain/purchase.rb)